user-journey-tests/page-objects/page.js

```diff
@@ -72,9 +88,29 @@ class Page {
     )
   }
 
+  async verifyFooter() {
+    await page.validateElementVisibleAndText(
+      this.getHelpFooterText(),
+      'If you need help with the service you can call 03000 200 301'
+    )
+    await page.validateElementVisibleAndText(
+      this.getPricacyFooterLink(),
+      'Privacy'
+    )
+    await page.validateElementVisibleAndText(
+      this.getCookiesFooterLink(),
+      'Cookies'
+    )
+    await page.validateElementVisibleAndText(
+      this.getAccessibilityFooterLink(),
+      'Accessibility statement'
+    )
+  }
```

Comment:
Can we also verify the hrefs point to the site as expected?  

I know we've tested that we can actually physically navigate to the destination page on the landing page.  That's definitely a good test to do once, but too expensive to do on every page. 

But validating the href would give us a lot of confidence without a lot of extra execution time & no change of page.